packages/node_modules/@ciscospark/internal-plugin-encryption/src/encryption.js

```diff
@@ -47,7 +47,7 @@ const Encryption = SparkPlugin.extend({
         .then((result) => result.plaintext.toString()));
   },
 
-  download(scr) {
+  download(scr, options) {
```

Comment:
Could you split this out to another commit?

Reply:
actually didn't need to add options to the encryption plugin. removed from PR and updated tests